packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import classnames from 'classnames';
+import ExpandCollapsePanel, {
+  ExpandCollapsePanelProps,
+  PanelTrigger
+} from '../ExpandCollapsePanel';
+import randomId from '../../utils/rndid';
+
+export interface AccordionTriggerProps
+  extends React.HTMLAttributes<HTMLButtonElement> {
+  className?: string;
+  children: React.ReactElement;
+}
+
+const AccordionTrigger = ({ children }: AccordionTriggerProps) => {
+  return <>{children}</>;
+};
+
+interface AccordionContentProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode | React.ReactNode[];
+  className: string;
+}
+
+const AccordionContent = ({
+  children,
+  className,
+  ...otherProps
+}: AccordionContentProps) => {
+  return (
+    <div className={classnames('Accordion__panel', className)} {...otherProps}>
+      {children}
+    </div>
+  );
+};
+
+interface AccordionProps extends ExpandCollapsePanelProps {
+  children: React.ReactNode;
+}
+
+const Accordion = ({ children, ...otherProps }: AccordionProps) => {
+  const [elementId, setElementId] = useState<string | null>(null);
+  const childrenArray = React.Children.toArray(children);
+  const trigger = childrenArray.find(
+    child => (child as React.ReactElement<any>).type === AccordionTrigger
+  );
+  const panel = childrenArray.find(
+    child => (child as React.ReactElement<any>).type === AccordionContent
+  );
+
+  useEffect(() => {
+    setElementId(randomId());
+    return;
+  }, []);
```

Comment:
Think I did it in the useEffect because it was re-calculating the random number repeatedly, but it might not be needed anymore after some of the other changes. Thank you for the reference and will try it out!